lib/modules/datasource/npm/get.ts

```diff
@@ -201,7 +201,7 @@ export async function getDependency(
     ) {
       return null;
     }
-    if (uri.host === 'registry.npmjs.org') {
+    if (uri.host === 'registry.npmjs.org' && err.message !== 'host-disabled') {
```

Comment:
should we check for disable before all other handling?

Reply:
Check new commit